src/Controls/src/Core/BindableObjectExtensions.cs

```diff
@@ -8,6 +8,22 @@ namespace Microsoft.Maui.Controls
 	/// <include file="../../docs/Microsoft.Maui.Controls/BindableObjectExtensions.xml" path="Type[@FullName='Microsoft.Maui.Controls.BindableObjectExtensions']/Docs/*" />
 	public static class BindableObjectExtensions
 	{
+		internal static void RefreshPropertyValue(this BindableObject self, BindableProperty property, object value)
+		{
+			var ctx = self.GetContext(property);
+			if (ctx?.Binding is not null)
+			{
+				// support bound properties
+				if (!ctx.Attributes.HasFlag(BindableObject.BindableContextAttributes.IsBeingSet))
+					ctx.Binding.Apply(false);
+			}
```

Comment:
@StephaneDelcroix is this the correct way to ask a bindable property to "refresh" or "re-get" the value.